cloudbuild-tag.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 Security Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
```

Comment:
Can you update the RELEASE.md explaining any difference that the pre-build image introduce for release purposes? Do we need to wait for a stable tag on the container?

Reply:
We don't have that level of automation. Basically, if we want to make a release, need to create a Git tag (say, x.y.z) and that'll automatically generate `scorecard-action:vx.y.z` image which we need to refer here in `action.yaml`. I'll update the notes with this.